src/pages/RoomMemberDetailsPage.tsx

```diff
@@ -47,7 +48,7 @@ function RoomMemberDetailsPage({report, route}: RoomMemberDetailsPagePageProps)
     const member = report?.participants?.[accountID];
     const details = personalDetails?.[accountID] ?? ({} as PersonalDetails);
     const fallbackIcon = details.fallbackIcon ?? '';
-    const displayName = details.displayName ?? '';
+    const displayName = formatPhoneNumber(PersonalDetailsUtils.getDisplayNameOrDefault(details));
```

Comment:
@jayeshmangwani i'll check that

Reply:
Haha, no problem! Keeping discussion in one place makes things faster, so 😃 .

Reply:
no it's needed @jayeshmangwani i did a test again
![Capture d’écran de 2024-12-02 18-08-41](https://github.com/user-attachments/assets/70eea1b7-a4a4-40c7-a87a-fae3881c235c)

Reply:
@Kalydosos I am adding the steps below; please try them on your side as well.

1. Add a user with a phone number.
2. Remove the `formatPhoneNumber` function from `RoomMemberDetailsPage`.

We can observe(console log) that the result of `PersonalDetailsUtils.getDisplayNameOrDefault(details)` is the same as `formatPhoneNumber(PersonalDetailsUtils.getDisplayNameOrDefault(details))`. This indicates that the formatPhoneNumber function is redundant in this context. Since we are already formatting the phone number, it doesn't need to be included here.

I’ve attached a video for reference. If anything is still unclear, feel free to let me know.

https://github.com/user-attachments/assets/cf35d0bd-ae44-484d-8be9-db445cec7f4d

Reply:
@jayeshmangwani maybe you had a usage of these accounts which end up setting the PersonalDetailsUtils.getDisplayNameOrDefault(details) as same value with the formatPhoneNumber value, as i have a different result as you can see. A good test will be to invite a new user with a new phone number. But anyway, you agree with me that we dont expect PersonalDetailsUtils.getDisplayNameOrDefault to act the same as formatPhoneNumber right ? And as using formatPhoneNumber seems to not harm the end result in both our cases i think we should keep it as not using it have the inverse impact in my test case. I'm saying our test cases  show that it's better to have it there instead of not to have it, as in my case the result is unsatisfying (the core issue of this ticket). What do you think ?

Reply:
> What do you think ?

On this PR `formatPhoneNumber` change is being used in many places throughout. If even one instance where `formatPhoneNumber` is not needed is included, it could lead to unnecessary regressions.

What I suggest is that we only use `formatPhoneNumber` in this PR at locations where the changes can be tested easily, and we can verify the outcome of those specific changes. Additionally, the two files `src/libs/ReportUtils.ts` and `src/libs/OptionsListUtils.ts` are used in several places. Any changes to these files should be made more cautiously to avoid potential issues.

Reply:
> you agree with me that we dont expect PersonalDetailsUtils.getDisplayNameOrDefault to act the same as formatPhoneNumber right ?

Agreed

Reply:
> And as using formatPhoneNumber seems to not harm the end result in both our cases i think we should keep it as not using it have the inverse impact in my test case

I don't agree with this point. My thinking is that if the text is already formatted the way we want, there's no need to format it again. Reapplying formatting might introduce issues or break functionality at some point, which we can't predict.

Reply:
@jayeshmangwani this is a test without formatPhoneNumber, as you can see, on the previous screens fromatting is correct but on roommemberdetails screen, even though +22951323930 is in the same region as me its details appear unformatted the same manner as +27440370877 which is in a different region. What do you suggess ? 

https://github.com/user-attachments/assets/0d809a82-a870-4445-842d-be03ebbf2425

Reply:
> What do you suggess ?
@Kalydosos Sorry, I’m a little confused— is this video from your branch or the main branch?

Reply:
it's my branch @jayeshmangwani

Reply:
i will do a new merge with main

Reply:
@jayeshmangwani i did a test again after the new merge and it's the result result as in the video

Reply:
@Kalydosos For me, it’s working fine both with and without using the `formatPhoneNumber`. I’m not sure why we’re getting different results.

Reply:
@Kalydosos Please leave the `RoomMemberDetailsPage.tsx` page as it is with the changes you’ve made; we’ll figure this out later. However, can you please check the other comments? I feel we shouldn’t make changes in a few places that I’ve highlighted.

Reply:
@jayeshmangwani I made a test again for this one and i still have the same result., any change on your side for this ?